.gitignore

```diff
@@ -51,7 +53,10 @@ public-api/wrapper
 public-api/.swagger-codegen/*
 public-api/src/generated/
 public-api/src/main/webapp/WEB-INF/appengine-web.xml
-
+public-api/src/main/resources/merged.yaml
+public-api/gradle
```

Comment:
Intellij generated these gradle wrapper files in my public-api dir. (I'm not sure if my settings were different whether this would have happened, but we don't want to include these files in the repo.)